code/API_definitions/Get DeviceLocation.yaml

```diff
@@ -0,0 +1,344 @@
+
+    Location:
+      type: object
+      description: Device location
+      required:
+        - lastLocationTime
+        - area
+      properties:
+        lastLocationTime: 
+          description: Last date and time when the device was localized (using ISO 8601 with timezone)
+          type: string
+          format: datetime
+        area:
+          $ref: '#/components/schemas/Area'
+        
+    Area:
+      type: object
+      required:
+        - areaType
+      properties:
+        areaType:
+          description: type of the area (like Circle)
+          type: string
+      discriminator: 
+          propertyName: areaType
+          mapping: 
+            Circle:  "#/components/schemas/Circle"
+
+    Circle:
+      description: Circular area
+      allOf: 
+        - $ref: "#/components/schemas/Area"
+        - type: object
+          required:
+            - center
+            - radius
+          properties:
+            center: 
+              $ref: "#/components/schemas/Point"
+            radius:
+              type: number
+              description: Distance from the center in meters
+              minimum: 1
+        
+    Point:
+      type: object
+      description: Coordinates (latitude, longitude) defining a location in a map
+      required: 
+        - latitude
+        - longitude
+      properties: 
+        latitude:
+          $ref: "#/components/schemas/Latitude"
+        longitude:
+          $ref: "#/components/schemas/Longitude"
+      example: 
+        latitude: 50.735851
+        longitude: 7.10066
+
+    Latitude:
+      description: Latitude component of a location
+      type: number
+      format: double
+      minimum: -90
+      maximum: 90
+  
+    Longitude:
+      description: Longitude component of location
+      type: number
+      format: double
+      minimum: -180
+      maximum: 180
+
+    ErrorInfo:
+      type: object
+      required:
+        - status
+        - code
+        - message
+      properties:
+        status:
+          type: integer
+          description: HTTP status code returned along with this error response
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+  responses:
+    RetrieveLocationBadRequest400:
+      description: |-
+        Problem with the client request. In addition to regular scenario of `INVALID_ARGUMENT`, another scenarios may exist:
+          - maxAge threshold cannot be satisfied
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorInfo'
+          examples:
+            InvalidArgument:
+              value:
+                status: 400
+                code: INVALID_ARGUMENT
+                message: 'Invalid input'
+            MaxAgeIssue:
+              value:
+                status: 400
+                code: LOCATION_RETRIEVAL.MAXAGE_INVALID_ARGUMENT

```

Comment:
If we follow also API_NAME.{CODE}  then it would be just LOCATION.MAXAGE_INVALID_ARGUMENT